### ISSUE TITLE
[Bug][SF][2.6.0][FlexibleColumnLayout][A11y][Expected reply Time: ASAP] Keyboard focus not visible on separator in High Contrast Mode

### ISSUE THREAD

**Issue by wilson-xu-sap** [SAP/ui5-webcomponents#10670]
### Describe the bug

<img width="1137" alt="Image" src="https://github.com/user-attachments/assets/ed2f6b03-7124-4d59-9ba0-85da52ac2b42" />

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/layouts-floorplans-flexiblecolumnlayout--docs&args=disableResizing:!true&globals=theme:sap_horizon_hcb

### Reproduction steps

Press tab to focus on sequence.

### Expected Behaviour

The keyboard focus should be visible on the separator in all modes.

### UI5 Web Components for React Version

2.6.0

### UI5 Web Components Version

2.6.0

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by petyabegovska** [comment]
Hello @SAP/ui5-webcomponents-topic-p,

The focus is not visible on the Splitter bar in Horizon HCB and HCW and can be reproduced in the dev samples also:
https://sap.github.io/ui5-webcomponents/nightly/packages/fiori/test/pages/FCL.html?sap-ui-theme=sap_horizon_hcb
https://sap.github.io/ui5-webcomponents/nightly/packages/fiori/test/pages/FCL.html?sap-ui-theme=sap_horizon_hcw

In SAPUI5 the dotted focus is visible: https://openui5nightly.hana.ondemand.com/test-resources/sap/f/demokit/sample/FlexibleColumnLayoutWithOneColumnStart/webapp/index.html?sap-ui-theme=sap_horizon_hcb&sap-ui-density=cozy&sap-ui-rtl=false#/detail/116/TwoColumnsMidExpanded

https://openui5nightly.hana.ondemand.com/test-resources/sap/f/demokit/sample/FlexibleColumnLayoutWithOneColumnStart/webapp/index.html?sap-ui-theme=sap_horizon_hcw&sap-ui-density=cozy&sap-ui-rtl=false#/detail/90/TwoColumnsMidExpanded

Please check with VD if needed which is the correct behaviour.

Best regards,
Petya

**Comment by wilson-xu-sap** [comment]
Hi @NakataCode Are you sure the bahavior now is correct?
https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/layouts-floorplans-flexiblecolumnlayout--docs&args=headerText:List%20with%20ListItemStandard;selectionMode:None;separators:All;growing:Button&globals=theme:sap_horizon_hcb
<img width="822" alt="Image" src="https://github.com/user-attachments/assets/d3949cb0-0746-4ab4-a9c4-f52745956d2e" />

**Comment by NakataCode** [comment]
Hello @wilson-xu-sap,

A fix for this is already included in our nightly build and is also available in the latest [release](https://github.com/SAP/ui5-webcomponents/releases/tag/v2.8.0-rc.2). It will be officially available in the next stable release.
